packages/graph-editor/src/registry/control.tsx

```diff
@@ -78,7 +79,10 @@ export const defaultControls = [
     matcher: (port: Port) => port.type.$id === ANY,
     component: AnyField,
   },
-
+  {
+    matcher: (port: Port) => port.type.type === 'array',
+    component: ArrayField,
+  },
   {
     matcher: variadicMatcher(NUMBER),
```

Comment:
None of these variadic matchers will match anymore since there is a higher priority matcher which matches all their use cases so we should delete them